tests/acceptance/NewsletterSendingErrorCest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Mailer\Mailer;
+use MailPoet\Test\DataFactories\Settings;
+
+require_once __DIR__ . '/../DataFactories/Settings.php';
+
+class NewsletterSendingErrorCest {
+  function generalErrorNotice(\AcceptanceTester $I) {
+    $I->wantTo('See proper sending error when sending failed');
+    $I->login();
+
+    $errorMessage = 'Error while sending email.';
+    $settings = new Settings();
+    $settings->withSendingMethod(Mailer::METHOD_SMTP);
+    $settings->withSendingError($errorMessage);
+
+    $I->amOnMailpoetPage('Emails');
+    $I->waitForElement('.mailpoet_notice.notice-error div');
+    $I->see('Sending has been paused due to a technical issue with SMTP: ' . $errorMessage, '.notice-error p');
+    $I->see('Check your sending method settings.', '.notice-error p');
+    $I->see('Resume sending', '.notice-error p');
+
+    $href = $I->grabAttributeFrom('//a[text()="sending method settings"]', 'href');
+    expect($href)->endsWith('page=mailpoet-settings#mta');
+
+    $I->click('Resume sending');
+    $I->waitForText('Sending has been resumed.');
+  }
+
+  function phpMailErrorNotice(\AcceptanceTester $I) {
+    $I->wantTo('See proper sending error when sending failed with PHPMail');
+    $I->login();
+
+    $errorMessage = 'Could not instantiate mail function. Unprocessed subscriber: (test <test@test.test>)';
+    $settings = new Settings();
+    $settings->withSendingMethod(Mailer::METHOD_PHPMAIL);
+    $settings->withSendingError($errorMessage);
+
+    $I->amOnMailpoetPage('Emails');
+    $I->waitForElement('.mailpoet_notice.notice-error div');
+    $I->see('Sending has been paused due to a technical issue with PHPMail: ' . $errorMessage, '.notice-error p');
+    $I->see('Please check your sending method configuration, you may need to consult with your hosting company.', '.notice-error p');
+    $I->see('The easy alternative is to send emails with MailPoet Sending Service instead, like thousand of other users do.', '.notice-error p');
```

Comment:
Hardcoding a lot of text to check seems like double work, e.g. same typos to correct. I don't know how to better handle this, maybe to use translation strings from templates, but this amounts to a task in itself for another time.

Reply:
I agree. I don't really know of a better way to do this at the moment. I tried to follow the requirements to check the notice exactly.